### ISSUE TITLE
gltf-transform cli cannot start with node16. Is there a certain node version which I have to use?

### ISSUE THREAD

**Issue by devjta** [donmccurdy/glTF-Transform#376]
Hi!

After running this command:  npm install --global @gltf-transform/cli
I would like to start the gltf-transform cli, but then this error appears:
```
ode:internal/modules/cjs/loader:1126
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
      ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /home/jota/.nvm/versions/node/v16.3.0/lib/node_modules/@gltf-transform/cli/node_modules/gl-matrix/dist/esm/vec3.js
require() of ES modules is not supported.
require() of /home/jota/.nvm/versions/node/v16.3.0/lib/node_modules/@gltf-transform/cli/node_modules/gl-matrix/dist/esm/vec3.js from /home/jota/.nvm/versions/node/v16.3.0/lib/node_modules/@gltf-transform/cli/node_modules/@gltf-transform/core/dist/core.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename vec3.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /home/jota/.nvm/versions/node/v16.3.0/lib/node_modules/@gltf-transform/cli/node_modules/gl-matrix/package.json.

```

**To Reproduce**
1. Install node16
2. Install gltf-transform cli with npm install --global @gltf-transform/cli
3. Try to run gltf-transform

**Expected behavior**
No error..

**Versions:**
 - Version: 0.12.15
 - Environment: Ubuntu 18.04

**Comment by pawel-marciniak** [comment]
I have exactly same issue while using it in script by importing things - https://github.com/donmccurdy/glTF-Transform/discussions/375 , also on Node 16.x. I wasted half a day on trying to make it work :/

**Comment by koeckc** [comment]
To make it even more weird ...
Yesterday i installed on my mac ... and it worked
node v14.17.3
MacOS BigSur
gltf-transform -v 0.12.15

Today on other machines not anymore .... really spooky

**Comment by devjta** [comment]
I found the issue... Its gl-matrix 3.4.0. It works fine, if I use gl-matrix 3.3.0.

Currently I just copied it into the project-folder/cli/node_modules. Then it starts fine.

**Comment by donmccurdy** [comment]
The integration tests check Node.js v12 and v14 so probably those are the safest options. I'll try adding v16 to the tests and see what happens – was v16 working before the gl-matrix dependency update for anyone?

It looks like gl-matrix has already gotten reports about this (https://github.com/toji/gl-matrix/issues/439) and are planning to revert 3.4.0 shortly. I'll see about switching this project's production dependencies to [`~` instead of `^`](https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json) as well.

**Comment by pawel-marciniak** [comment]
Ok, so they published new version for gl-matrix (v3.4.1), but the problem remains, only solution for now is to revert to gl-matrix@3.3.0

**Comment by donmccurdy** [comment]
I've just published v0.12.16, pinned to gl-matrix@~3.3.0. Hopefully that fixes the regression, but let me know if not! 

Have also added integration test coverage for Node.js v16, which seems to be working fine.

**Comment by devjta** [comment]
Thx for the fast fix!
